From 2a062bc0b9d6877590a46acc32ff0c735fb1ea18 Mon Sep 17 00:00:00 2001
From: Ken McGrady <ken.mcgrady@snowflake.com>
Date: Tue, 22 Aug 2023 12:15:12 -0700
Subject: [PATCH] Fix pydeck version

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 72e5bdd..7d77092 100644
--- a/setup.py
+++ b/setup.py
@@ -71,7 +71,7 @@ INSTALL_REQUIRES = [
 # `pip install streamlit` or `conda install -c conda-forge streamlit`)
 SNOWPARK_CONDA_EXCLUDED_DEPENDENCIES = [
     "gitpython>=3.0.7, <4, !=3.1.19",
-    "pydeck>=0.8, <1",
+    "pydeck>=0.8.0b4, <1",
     # Tornado 6.0.3 was the current Tornado version when Python 3.8, our earliest supported Python version,
     # was released (Oct 14, 2019).
     "tornado>=6.0.3, <7",
-- 
2.37.1

